src/content/fr/2019/markup.md

```diff
@@ -0,0 +1,283 @@
+    </a>
+    <div id="fig5-description" class="visually-hidden">Ce graphique représente le nombre moyen d'éléments est une courbe en cloche autour des 30 éléments marqués, telle qu'utilisée par 308 168 milliers de sites.</div>
+  <figcaption id="fig5-caption">Figure 5. Histogramme des types d'éléments par page à partir de 2019.</figcaption>
+</figure>
+
+Nous pouvons voir sur les deux que nombre moyen de types d'éléments par page a augmentés, ainsi que le nombre maximal d'éléments uniques que nous rencontrons.
+
+## Éléments personnalisés
+
+La plupart des éléments que nous avons enregistrés sont personnalisés (tout simplement «non standard»), mais il peut être un peu difficile de déterminer quels sont des éléments personnalisés ou non. Écrit dans une spécification où proposition quelque part est, actuellement, un certain nombre d'éléments. Pour ce but, nous avons considéré 244 éléments comme standard (cependant, certains d’entre eux sont obsolètes ou non pris en charge):
+
+* 145 Éléments de HTML
+* 68 Éléments de SVG
+* 31 Éléments de MathML
+
+En pratique, nous n'avons rencontré que 214 d'entre eux:
+
+* 137 Éléments de HTML
+* 54 Éléments de SVG
+* 23 Éléments de MathML
+
+Dans l'ensemble de données de bureau, nous avons collecté des données pour les top 4 834 éléments non standard rencontrés. Parmi ceux-ci:
+
+* 155 (3%) sont identifiables comme des balisages très probable ou des erreurs d'échappement (ils contiennent des caractères dans le nom de balise analysé qui implique que le balisage est cassé)
+* 341 (7%) utilisent un espace de nom de style XML de type deux-points (bien que, comme HTML, ils n'utilisent pas d'espaces de nom XML réels)
+* 3 207 (66%) sont des noms d'élément personnalisé valides
+* 1 211 (25%) sont dans l'espace de noms global (non standard, sans tiret ni deux-points)
+    * 216 de celles-ci ont été marquées comme * possibles * des fautes de frappe car elles sont plus longs que 2 caractères et ont une distance de Levenshtein de 1 à partir d'un nom d'élément standard comme `<cript>`,`<spsn>` ou `<artice>`. Certains d'entre eux (comme `<jdiv>`), Cependant, sont certainement intentionnelle.
+
+En outre, 15% des pages d'ordinateur et 16% des pages de mobile contiennent des éléments obsolètes.
+
+<aside class="note">Remarque: cela est très probablement dû à l'utilisation de produits plutôt qu'au fait que des auteurs individuels continuent à créer manuellement ce balisage.</aside>
+
+<figure id="fig6">
+    <a href="/static/images/2019/03_Markup/fig6.png">
+      <img src="/static/images/2019/03_Markup/fig6.png" alt="Figure 6. Most frequently used deprecated elements." aria-labelledby="fig6-caption" aria-describedby="fig6-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTbHgqcSepZye6DrCTpifFAUYxKT1hEO56585awyMips8oiPMLYu20GETuIE8mALkm814ObJyktEe2P/pubchart?oid=1304237557&amp;format=interactive">
```

Comment:
```suggestion
      <img src="/static/images/2019/03_Markup/fig6.png" alt="Figure 6. Éléments déconseillés les plus fréquemment utilisés." aria-labelledby="fig6-caption" aria-describedby="fig6-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTbHgqcSepZye6DrCTpifFAUYxKT1hEO56585awyMips8oiPMLYu20GETuIE8mALkm814ObJyktEe2P/pubchart?oid=1304237557&amp;format=interactive">
```

Reply:
sir @bazzadp can you explain a little bit more  ???

Reply:
Looks like you missed translating the `alt` attribute. So I copied your translation of the `figcaption` since they are the same.

![image](https://user-images.githubusercontent.com/10931297/69740326-1f096300-1131-11ea-8eb1-c12c4eab6b50.png)